gtwrap/interface_parser/tokens.py

```diff
@@ -46,3 +46,48 @@
         "float",
     ],
 )
+
+OPERATOR = Or(
+    map(
+        Literal,
+        [
+            '+',  # __add__, __pos__
+            '-',  # __sub__, __neg__
+            '*',  # __mul__
+            '/',  # __truediv__
+            '%',  # __mod__
+            '^',  # __xor__
+            '&',  # __and__
+            '|',  # __or__
+            # '~',  # __invert__
+            # '=',  # Not supported
+            '+=',  # __iadd__
+            '-=',  # __isub__
+            '*=',  # __imul__
+            '/=',  # __itruediv__
+            '%=',  # __imod__
+            '^=',  # __ixor__
+            '&=',  # __iand__
+            '|=',  # __ior__
+            '<<',  # __lshift__
+            '<<=',  # __ilshift__
+            '>>',  # __rshift__
+            '>>=',  # __irshift__
+            '==',  # __eq__
+            '!=',  # __ne__
+            '<',  # __lt__
+            '>',  # __gt__
+            '<=',  # __le__
+            '>=',  # __ge__
+            # '!',  # `not`
+            # '&&',  # `and`
+            # '||',  # `or`
+            '++',  # Not supported
```

Comment:
Remove them if not supported?

Reply:
Good point. I was wondering if we should leave them in as "documentation" for whomever looks at this next?